src/state/settings-form/reducer.js

```diff
@@ -0,0 +1,12 @@
+import { GET_OPTIONS } from './types';
```

Comment:
Please change the reducer name in rootReducer (options is way too generic), and then name the directory after the new name (settings-form is not a good choice, as the form state is handled by redux-form)